controllers/cluster_controller.go

```diff
@@ -554,3 +560,15 @@ func (r *ClusterReconciler) setBundlesRef(ctx context.Context, clus *anywherev1.
 	clus.Spec.BundlesRef = mgmtCluster.Spec.BundlesRef
 	return nil
 }
+
+func (r *ClusterReconciler) setDefaultEksaVersion(ctx context.Context, clus *anywherev1.Cluster) error {
+	mgmtCluster := &anywherev1.Cluster{}
+	if err := r.client.Get(ctx, types.NamespacedName{Name: clus.ManagedBy(), Namespace: clus.Namespace}, mgmtCluster); err != nil {
```

Comment:
It looks like we can refactor this call out to make it easier not have to replicate this business logic and instead just call a function and have it return as expected, like `ValidateManagementClusterExists`